docs/source/notebooks/Delineation_workflow.ipynb

```diff
@@ -22,11 +22,19 @@
     "import matplotlib.pyplot as plt\n",
     "import geopandas as gpd\n",
     "import json\n",
-    "import os\n",
-    "\n",
-    "\n",
+    "import os"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
     "# Set environment variable RAVEN_WPS_URL to \"http://localhost:9099\" to run on the default local server\n",
-    "url = os.environ.get(\"RAVEN_WPS_URL\", \"https://pavics.ouranos.ca/twitcher/ows/proxy/raven/wps\")\n",
+    "# url = os.environ.get(\"RAVEN_WPS_URL\", \"https://pavics.ouranos.ca/twitcher/ows/proxy/raven/wps\")\n",
+    "url = os.environ.get(\"RAVEN_WPS_URL\", \"http://localhost:9099\")\n",
```

Comment:
The default Raven URL should have been the one on PAVICS, not the localhost one !